package.json

```diff
@@ -138,6 +140,16 @@
 				"title": "Drop"
 			},
 			{
+				"category": "DocumentDB",
+				"command": "cosmosDB.dropDocDBDatabase",
+				"title": "Drop Database"
+			},
+			{
+				"category": "DocumentDB",
+				"command": "cosmosDB.dropDocDBCollection",
+				"title": "Drop Collection"
```

Comment:
The portal uses 'Delete' instead of 'Drop'. Can you change all occurrences to Delete?

(Including Mongo - I'm not sure why it was 'Drop' in the first place)

Reply:
I think Drop comes from the sequel terminology of dropping a table.